src/Rings/msolve/msolve.jl

```diff
@@ -0,0 +1,205 @@
+import msolve_jll: libmsolve
+
+export msolve
+
+@doc Markdown.doc"""
+    function get_rational_parametrization(nr::Int32, lens::Array{Int32,1}, cfs::Ptr{BigInt})
+
+Construct the rational parametrization of the solution set computed via msolve.
+
+**Note**: This is an internal function and should only be used inside `msolve()`.
+"""
+function get_rational_parametrization(
+        nr::Int32,
+        lens::Array{Int32,1},
```

Comment:
Minor thing: in general we've switched to using `Vector{T}` instead of `Array{T,1}` (and `Matrix{T}` instead of `Array{T,2}`), as this is also what Julia "recommends" these days. However, that's of course not a blocker for this PR; I just mention it to make sure you are aware.
```suggestion
        lens::Vector{Int32},
```